src/py/_pyodide/_core_docs.py

```diff
@@ -124,6 +124,36 @@ def to_bytes(self) -> bytes:
         an ArrayBuffer view.
         """
 
+    def to_file(self, fd: int):
+        """Writes the entire buffer to a file descriptor.
+
+        Will write the entire contents of the buffer to the current position of
+        the file descriptor.
+
+        Present only if the wrapped Javascript object is an ArrayBuffer or an
+        ArrayBuffer view.
+        """
+
+    def from_file(self, fd: int):
+        """Reads from a file descriptor into the buffer.
+
+        Will try to read a chunk of data of size the length of the buffer from
+        the current position of the file descriptor.
+
+        Present only if the wrapped Javascript object is an ArrayBuffer or an
+        ArrayBuffer view.
```

Comment:
Also an example would be good to really understand how to use this, something like,
```suggestion
        ArrayBuffer view.


Example
------------
>>> from js import Uint8Array
>>> # the JsProxy need to be pre-allocated
>>> x = Uint8Array.new(range(10))
>>> with open('file.bin', 'w') as fh:
...    x.read_file(fh.fileno())
which is equivalent to,
>>> with open('file.bin', 'w') as fh:
...    chunk = fh.read(size=x.byteLength)
...    # not sure how to assign this result to x buffer
but the latter is less efficient and makes more memory copies.
```